### ISSUE TITLE
MOSS training: request example dataset files for instruction (emotion) field

### ISSUE THREAD

**Issue by krisheeetu** [diodiogod/TTS-Audio-Suite#361]
I've read MOSS_TRAINING_GUIDE.md and MOSS_TTS_PROMPT_FIELDS_GUIDE.md, but I'm still unsure how to correctly format the `instruction` field for emotion-conditioned training on laion/moss-tts-v1.5-8b-voice-acting.

Could you share a small example dataset (a few rows of manifest.jsonl, or example instruction_lines values) showing a working instruction-conditioned training setup?

Also, does the folder auto-import on MOSS Dataset Prep support `instruction` at all,
or is a JSONL manifest via MOSS Dataset Rows required to use it?

A minimal sample (3-5 rows) in the docs or example_workflows would help a lot. Thanks!

**Comment by diodiogod** [comment]
I only implemented the MOSS training integration; I did not create the upstream training method or the LAION Voice Acting fine-tune. My own test was a LoRA for one specific voice, without instructions, so I cannot claim a tested best practice for instruction-conditioned training.

From checking the code:

- `instruction` is preserved in the JSONL and passed directly to the official MOSS `processor.build_user_message(...)` for each training row.
- `MOSS Dataset Rows` supports it through `instruction_lines`: one line per clip, in the same order as `text_lines`.
- Folder auto-import does **not** support instructions. It only converts matching `clip.wav` + `clip.txt` files into `audio + text` rows. To use per-clip instructions, use `MOSS Dataset Rows` or provide your own JSONL manifest.

Structurally, a manifest can look like this:

```jsonl
{"audio":"./clips/001.wav","text":"I cannot believe you did that.","instruction":"Angry and confrontational, speaking loudly.","language":"English"}
{"audio":"./clips/002.wav","text":"Everything is going to be all right.","instruction":"Warm, gentle, and reassuring.","language":"English"}
{"audio":"./clips/003.wav","text":"Did you hear something downstairs?","instruction":"Fearful whisper, tense and hesitant.","language":"English"}
```

With `MOSS Dataset Rows`, put the three transcripts in `text_lines` and the three matching descriptions in `instruction_lines`.

However, these rows demonstrate the supported format, not a recipe I have personally validated. The official OpenMOSS trainer maps `instruction` into the prompt in the same way, but its clearest `text + instruction` training examples are for VoiceGenerator rather than a documented emotion-training recipe for MOSS-TTS v1.5. The LAION model card says performance direction, emotion, character, pacing, and similar cues belong in `instruction`, but it does not publish the actual training manifest or label format used for that checkpoint.

So you will ultimately need to experiment with wording and dataset consistency. I would start with short, consistent natural-language descriptions, use the same vocabulary for recurring emotions/styles, set `language`, and compare a small LoRA trained with instructions against a plain `audio + transcript` baseline before committing to a large run.
